packages/flutter/lib/src/animation/tween.dart

```diff
@@ -108,9 +119,11 @@ class IntTween extends Tween<int> {
   int lerp(double t) => (begin + (end - begin) * t).round();
 }
 
+/// Transforms the value of the given animation by the given curve.
 class CurveTween extends Animatable<double> {
```

Comment:
CurveTween is named to be consistent with ColorTween and RectTween.  CurveAnimation would be an animation of a curve.  CurvedAnimation is an animation that has been curved.

Really CurveTween is the wrong name for that class because it's not a tween!

Another tough one is ReverseAnimation versus ReversedAnimation.